standard/PortalStatistics.lua

```diff
@@ -0,0 +1,1138 @@
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Currency = require('Module:Currency')
+local DateExt = require('Module:Date/Ext')
+local Info = require('Module:Info')
+local LeagueIcon = require('Module:LeagueIcon')
+local Lpdb = require('Module:Lpdb')
+local Math = require('Module:Math')
+local Medal = require('Module:Medal')
+local Operator = require('Module:Operator')
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Tier = require('Module:Tier/Custom')
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+local OpponentDisplay = OpponentLibraries.OpponentDisplay
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local Count = require('Module:Count/dev')
+local PieChart = require('Module:Tournaments breakdown pie chart/dev')
+
+local CURRENT_YEAR = tonumber(os.date('%Y'))
+local DATE = os.date('%F')
+local TIMESTAMP = DateExt.readTimestamp(DATE)
+local DEFAULT_ALLOWED_PLACES = '1,2,3,1-2,2-3,2-4,3-4'
+local DEFAULT_ROUND_PRECISION = Info.defaultRoundPrecision or 2
+local LANG = mw.getContentLanguage()
+local LIST_OF_MODES = 'solo, team, other'
+local LIST_OF_GAMES = Array.map(Array.extractValues(Info.games, Table.iter.spairs), function(value)
+	return value['name']
+end)
+
+
+local StatisticsPortal = {}
+
+
+--[[
+Section: Chart Entry Functions
+]] --
+
+
+function StatisticsPortal.gameEarningsChart(args)
+
+	args = args or {}
+	args.customInputs = String.isNotEmpty(args.customInputs) and
```

Comment:
imo move this to the config function to deduplicate stuff
just pass the default values as a param to the config function